core/arch/arm/pta/system.c

```diff
@@ -35,6 +38,13 @@ static TEE_Result system_rng_reseed(struct tee_ta_session *s __unused,
 
 	entropy_sz = MIN(entropy_sz, MAX_ENTROPY_IN);
 
+	utc = to_user_ta_ctx(s->ctx);
+	res = tee_mmu_check_access_rights(utc, TEE_MEMORY_ACCESS_READ |
```

Comment:
This is already done at:
https://github.com/OP-TEE/optee_os/blob/4d22155ce81bb04e3bcf171193eabf7b7e63f8ca/core/tee/tee_svc.c#L520-L521

Reply:
OK, I will drop this patch. Then I suppose this is not needed either?
https://github.com/OP-TEE/optee_os/blob/master/core/arch/arm/pta/gprof.c#L92-L95

(to be fixed in a separate patch)

Reply:
Yes, that's how I understand it.